hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -193,6 +195,8 @@ public class OmMetadataManagerImpl implements OMMetadataManager,
    * |----------------------------------------------------------------------|
    * |  snapshotInfoTable | /volume/bucket/snapshotName -> SnapshotInfo     |
    * |----------------------------------------------------------------------|
+   * |  renamedKeyTable   |  objectID -> RepeatedString                     |
```

Comment:
In order for efficient table cleanup when creating a snapshot, we need a slight improvement here that appends the volume and bucket name before the key object ID.

```suggestion
   * |  renamedKeyTable   |  /volumeName/bucketName/objectID -> OmKeyRenameInfo                     |
```